lsr_role2collection/collection_readme.md

```diff
@@ -13,8 +13,8 @@ NOTE: Some roles are not supported in RHEL6 and RHEL7. For more details about th
 
 ## Dependencies
 
-The following dependency is required for the Ansible Controller:
-* jmespath
+The following dependencies are required for the Ansible Controller:
+%dependencies%
```

Comment:
After thinking about this some more - I don't think we need a `Dependencies` section.

If the user is installing from Galaxy or Automation Hub, the user should know to look in `requirements.txt` and/or `bindep.txt` if the tooling doesn't use those files automatically (e.g. if you do `ansible-galaxy collection install fedora.system_roles` does it know to look in `requirements.txt` and ensure those dependencies?  If not, then it is the responsibility of the user).

If installing from RPM, the dependencies should automatically be installed - in that case, we should tell the user that.

Perhaps this wording:
```
## Dependencies

If installing from RPM, the dependencies will be installed with the package.  Otherwise, the dependencies are listed
in `requirements.txt` and/or `bindep.txt`.
```